packages/components/navbar/src/NavbarSwitcher/NavbarSwitcher.styles.ts

```diff
@@ -33,23 +32,22 @@ export const getNavbarSwitcherStyles = () => ({
         display: 'none',
       },
       '&:hover li': {
-        backgroundColor: hexToRGBA(tokens.colorWhite, 0.15),
+        backgroundColor: '#f2f4f6', // calculated hex color to avoid opaque color blending
```

Comment:
I'm trying to understand what was happening here, can you explain? Were two layers positioned on top of each other or something, which would cause issues with the opacity? Can we explain how this colour was calculated, so we can easily do it again when the colours change?

Reply:
The individual li elements are overlapping as we do not longer have the gap between them. I also could not put the color on the wrapping element as it would show a fully wrong width (because of the overlapping child elements). I tried to adjust the clipped path that was on them before that created the white halo. But i was not able to get it really working without looking wonky (is there a tool that calculates the right clip path for you?)

Anyhow, this is why I decided to not use the opacity. I simply did a color picker from the desired gray tone rendered.